src/pyaro_readers/eeareader/eeadownloader.py

```diff
@@ -0,0 +1,295 @@
+import requests
+
+import polars as pl
+import typer
+from typing_extensions import Annotated
+
+from pathlib import Path
+
+
+import toml
+
+
+from tqdm import tqdm
+
+
+app = typer.Typer()
+
+
+DATABASES = dict(
+    NRT=1,
+    VERIFIED=2,
+    HISTORICAL=3,
+)
+
+
+class EEADownloader:
+    BASE_URL = "https://eeadmz1-downloads-api-appservice.azurewebsites.net/"
+    ENDPOINT = "ParquetFile/"
+    URL_ENDPOINT = "urls"
+    URL_POLLUTANT = "http://dd.eionet.europa.eu/vocabulary/aq/pollutant/"
+
+    METADATFILE = Path(__file__).parent / "metadata.csv"
```

Comment:
this is not the station list, it is the pollutant definition: notation, numeric IDs, units &c.

IMO it should be part of packaging, as it will seldom change.

Other option is to cache the file in `f"~/.cache/{__package__}"`.
We can use [pooch] instead of requests, as it will also handle cache invalidation.

[pooch]: https://www.fatiando.org/pooch/latest/retrieve.html

Reply:
@avaldebe metadata.csv look very much like a station list and should therefore be stored together with the data:
```Sampling Point Id, Country, Air Quality Station EoI Code, Longitude, Latitude, Altitude, Air Quality Station Area, Air Quality Station Type, 
SPO-AD0942A-0001, Andorra, AD0942A, 1.539100000, 42.5097, 1080.0, urban, background, 
SPO-AD0942A-0005, Andorra, AD0942A, 1.539100000, 42.5097, 1080.0, urban, background, 
SPO-AD0942A-0007, Andorra, AD0942A, 1.539100000, 42.5097, 1080.0, urban, background, 
SPO-AD0942A-0008, Andorra, AD0942A, 1.539100000, 42.5097, 1080.0, urban, background, 
SPO-AD0942A-0009, Andorra, AD0942A, 1.539100000, 42.5097, 1080.0, urban, background, 
SPO-AD0942A-0010, Andorra, AD0942A, 1.539100000, 42.5097, 1080.0, urban, background, 
SPO-AD0942A-0038, Andorra, AD0942A, 1.539100000, 42.5097, 1080.0, urban, background, 
SPO-AD0942A-6001, Andorra, AD0942A, 1.539100000, 42.5097, 1080.0, urban, background, 
SPO-AD0944A-0007, Andorra, AD0944A, 1.565300000, 42.5169, 1637.0, rural, background, ```

The pollution definition list is in the toml-file, which is ok to store in the package.